Add new map The golden gate bridge by liverpool and more

LuismiInTheZone opened the pull request:
I have been working and talking with liverpool to continue the project he created in the past in the PR #218 , and some things have been changed from those comments too, for example, deleting the castles. Lets hope this gets into main CTF since is a very cool map at least for me. Good job guys.

lazylier commented:
I think this map looks good, but it looks like when it was saved some extra area around the outside of the glass was included that wasn't meant to be part of the map. That ought to be removed.

lazylier commented:
This map looks good, but maybe it should be resaved?

HobbitPower commented:
@LuismiInTheZone Not really a review, just curious: what did you add to the map?
The bridge and ships don't look bad. The secret rooms and staircases look a bit out of place.
I'll take a closer look at it soon.

LuismiInTheZone commented:
> @LuismiInTheZone Not really a review, just curious: what did you add to the map? The bridge and ships don't look bad. The secret rooms and staircases look a bit out of place. I'll take a closer look at it soon.
> 
> P.S.: I would edit the hint and shorten the additional informations a bit. `You can find ores in the secret stone rooms on the sides of the map` or `Ores can be found in the secret stone rooms on the western side of the map.` would work better imo. Or you can find an even better wording yourself.

I removed some medieval towers that were on the sides of the bases, they didn't match much in the map, I also fixed the stone rooms with minerals since wasn't organized well and one side had more minerals than the other. Another thing was a part of the corner of the outer barrier that was with a hole (but I think it was my fault when changing the map from v2 to v3). That's the part of building the map, then in the .conf file I edited the texts and changed some things. I wasn't going to add myself to the .conf file as an author, but liverpool said that wanted me to be there, so I added my name. The secret rooms and stairs were Elvis26's original idea, I can talk to liverpool if those things need to be changed.

LuismiInTheZone commented:
> @LuismiInTheZone Not really a review, just curious: what did you add to the map? The bridge and ships don't look bad. The secret rooms and staircases look a bit out of place. I'll take a closer look at it soon.
> 
> P.S.: I would edit the hint and shorten the additional informations a bit. `You can find ores in the secret stone rooms on the sides of the map` or `Ores can be found in the secret stone rooms on the western side of the map.` would work better imo. Or you can find an even better wording yourself.

I think is a bit difficult to shorten more this, but... I can put ''Look to the secret stone rooms on the sides of the map to find ores.''. Before there was ''Look to the secret stone rooms from the sides of the map, you'll find ores'' (I also saw it was gramatically wrong,  you can't use ''from'')

LuismiInTheZone commented on golden_gate_bridge/map.conf:
```diff
@@ -0,0 +1,21 @@
+author = liverpool, Elvis26, Luis_Mi and DiamondPlane
+hint = Look to the secret stone rooms on the sides of the map to find ores.
+license = CC BY-SA 4.0
+others = Thanks to AnthonyDe for helping fixing some holes and Liverpool that helped to build the bridge and the road.
```
I also changed the ''others'' part, seems I forgot to change this part when I uploaded this, it was a bit messy

src4026 commented:
> And... I think I still have to convert it to the modern version of CTF?

Yeah

LuismiInTheZone commented:
Ok done, everything is now ok to continue testing

Themostrandom commented:
Hello, I suggest adding ladders to the initial stuff, and correcting some inconsistencies on the bridge: the light poles are breakable. Your map is very beautiful, but I think that adding a skybox would make it even more beautiful.
![screenshot_20240410_135014](https://github.com/MT-CTF/maps/assets/165834755/098e4e92-6464-4567-bcdb-613ad6fe1626)
![screenshot_20240410_135010](https://github.com/MT-CTF/maps/assets/165834755/0c1ad2ec-fa96-4b65-b189-2234641c8750)
![screenshot_20240410_135000](https://github.com/MT-CTF/maps/assets/165834755/e81dda92-6dda-4228-b42e-142452b24b74)
![screenshot_20240410_134955](https://github.com/MT-CTF/maps/assets/165834755/73250c5a-844a-46c4-b6af-a90b259a9d1a)
![screenshot_20240410_134345](https://github.com/MT-CTF/maps/assets/165834755/b394059e-e96c-4f2e-a8bf-04ec0fd8ffb4)

LuismiInTheZone commented:
> Hello, I suggest adding ladders to the initial stuff, and correcting some inconsistencies on the bridge: the light poles are breakable. Your map is very beautiful, but I think that adding a skybox would make it even more beautiful. I'm looking forward to playing on your map. ![screenshot_20240410_135014](https://private-user-images.githubusercontent.com/165834755/321226966-098e4e92-6464-4567-bcdb-613ad6fe1626.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjUzMDIsIm5iZiI6MTcxMjc2NTAwMiwicGF0aCI6Ii8xNjU4MzQ3NTUvMzIxMjI2OTY2LTA5OGU0ZTkyLTY0NjQtNDU2Ny1iY2RiLTYxM2FkNmZlMTYyNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxMFQxNjAzMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ODdlZWY2MjE4NTJjYjZmZWQ0NTMwMDU2YWI1MTMyMDFmMmRkY2E3ZTVmNTQ4ZTI3MDRlOTM5OTYwNzIzMDVkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.t0gy291wLrpPD468UtJT8Ce9n41F1M3ccIMi5HklQ8I) ![screenshot_20240410_135010](https://private-user-images.githubusercontent.com/165834755/321226979-0c1ad2ec-fa96-4b65-b189-2234641c8750.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjUzMDIsIm5iZiI6MTcxMjc2NTAwMiwicGF0aCI6Ii8xNjU4MzQ3NTUvMzIxMjI2OTc5LTBjMWFkMmVjLWZhOTYtNGI2NS1iMTg5LTIyMzQ2NDFjODc1MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxMFQxNjAzMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZWQ4OTI5OGEwZjE0OGJlOTgxMGMyMTk4YzM4YzgxZjdiNmYzMjUyYTYzY2U0OTAzMjQzNDQ0ZWM2YjkxZjE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.MUWDB8ooPuYZ_njCnMOXjiHYcibMJipXbySRO2_1Qh0) ![screenshot_20240410_135000](https://private-user-images.githubusercontent.com/165834755/321226987-e81dda92-6dda-4228-b42e-142452b24b74.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjUzMDIsIm5iZiI6MTcxMjc2NTAwMiwicGF0aCI6Ii8xNjU4MzQ3NTUvMzIxMjI2OTg3LWU4MWRkYTkyLTZkZGEtNDIyOC1iNDJlLTE0MjQ1MmIyNGI3NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxMFQxNjAzMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wY2FkYTkzYjI2ODU0MmIzNzdhZTZlOGU5ZjFmYmRiMzQ2Nzg0NWUyZjgxNjk2MTUyMTljZWQ0NTkyY2Y2ZDc5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.18x0Y5uenGPSOjTJrN55mHvJoIBLVUl3U--1_K1bEoM) ![screenshot_20240410_134955](https://private-user-images.githubusercontent.com/165834755/321227002-73250c5a-844a-46c4-b6af-a90b259a9d1a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjUzMDIsIm5iZiI6MTcxMjc2NTAwMiwicGF0aCI6Ii8xNjU4MzQ3NTUvMzIxMjI3MDAyLTczMjUwYzVhLTg0NGEtNDZjNC1iNmFmLWE5MGIyNTlhOWQxYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxMFQxNjAzMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZTgwZmQyYjg4YzI2MzcxMWUyODhhODA5NjhmZWYxNjM4NGQzMTczNTc2MWZhMTMwNjcwMGM1MDhiMTY1NTM0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.RjAac1QHNt85DX-MzGNjZKO0fRVpC1sRk8pnZamJcTM) ![screenshot_20240410_134345](https://private-user-images.githubusercontent.com/165834755/321227006-b394059e-e96c-4f2e-a8bf-04ec0fd8ffb4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjUzMDIsIm5iZiI6MTcxMjc2NTAwMiwicGF0aCI6Ii8xNjU4MzQ3NTUvMzIxMjI3MDA2LWIzOTQwNTllLWU5NmMtNGYyZS1hOGJmLTA0ZWMwZmQ4ZmZiNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxMFQxNjAzMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNjE2MTMyZDhhMjVmZDU4NzI3MDUzNjcyMWIwNGRmNzRiNWRjYzdlOGI3NDQxMzY0NzIwMDBjYTA5Y2I0MGE4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.a8qd5K7XPc2lA4E3kR5tih3RQ3f0rkkxo1syqJmi5xo)

Hey! Thanks for testing it! I will see what can I do with the destructible and undestructible blocks, I really don't know if are on purpous or not, but maybe would be nice to make them undestructible, or at least some parts of the cars I found, but anyways, I'll talk with liverpool about it. The skybox seems nice specially with the summer one, but I think we'll probably won't add skybox. In case we add it, I'll tell in the commits. And for the ladders same as before, will talk with the team to see if can be good, if not, maybe an idea is putting as initial blocks.

LuismiInTheZone commented:
Any more suggestion that might be good to the map are always accepted (and hope this gets merged soon)

MoreKat commented on golden_gate_bridge/map.conf:
```diff
@@ -0,0 +1,21 @@
+size = return {y=140,x=220,z=220}
+map_version = 2
+enabled = true
+name = The golden gate bridge
```
Please change the name to `The Golden Gate Bridge` (with capital letters)

MoreKat commented on golden_gate_bridge/map.conf:
```diff
@@ -0,0 +1,21 @@
+name = The golden gate bridge
+phys_gravity = 1
+initial_stuff = return {"default:cobble 50"}
+author = Elvis26, liverpool, and DiamondPlane
```
Fix grammar: `author = Elvis26, liverpool and DiamondPlane`

MoreKat replied:
Fix grammar: `author = Elvis26, liverpool and DiamondPlane`

MoreKat commented on golden_gate_bridge/map.conf:
```diff
@@ -0,0 +1,21 @@
+phys_gravity = 1
+initial_stuff = return {"default:cobble 50"}
+author = Elvis26, liverpool, and DiamondPlane
+hint = Look to the secret stone rooms on the sides of the map to find ores.
```
Fix grammar: hint = `Look in the secret stone rooms on the sides of the map to find ores.`

MoreKat replied:
Fix grammar: hint = `Look in the secret stone rooms on the sides of the map to find ores.`

MoreKat reviewed (commented):
I like the map, I think the landscape is interesting. I just don't understand why there is a sneaky tunnel on blue's side ending directly behind their flag and this isn't the case on red's side, this seems unfair to me.

LuismiInTheZone commented:
Hey @MoreKat , hope you're doing great and thanks for reviewing the map! The .conf file is so messed up, I don't know what even happened that I deleted my own name from creators, anyways, I'm going to upload the .conf file again. 

I saw the tunnel you say, honestly I didn't realise about that, this was originally a project from Elvis26, and I fixed all the errors that had the map and fixes that could be good, told by liverpool or thoughts that I had. So since Elvis26 is gone I don't think we are going to build another tunnel, so I'll remove it later when I have some free time (aproximately I guess I'll update the .mts file tomorrow)

LuismiInTheZone commented:
Ok nevermind already updated

HobbitPower commented:
@LuismiInTheZone I think you have to re-save it with the newest CTF Version. It says `map_version = 2` instead of `map_version = 3`. Then it can be merged.

LuismiInTheZone commented:
@HobbitPower done 👍

HobbitPower reviewed (approved):
Double-checked the barriers, should be no problem this time.
Fair teams over all. I like the bridge and the ships, solid map. Approved
